test/lib.sh

```diff
@@ -2,7 +2,7 @@
 
 declare project_root="${0%/*}/.."
 declare ringpop_common_dir="${0%/*}/ringpop-common"
-declare ringpop_common_branch="master"
+declare ringpop_common_branch="menno/self-eviction"
```

Comment:
Yes, otherwise we will have the same problems as earlier where `dev` is not tested against `ringpop-common`'s `master` branch.

**Note to self:** DO NOT FORGET THIS TIME